lib/components/viewers/trip-viewer.js

```diff
@@ -32,91 +33,100 @@ class TripViewer extends Component {
+                    values={{
+                      routeLongName: tripData.route.longName,
+                      routeShortName: tripData.route.shortName,
+                      strong: Strong
+                    }}
+                  />
+                )}
               </div>
 
               {/* Wheelchair/bike accessibility badges, if applicable */}
               <h4>
-                {tripData.wheelchairAccessible === 1 &&
-                  <Label bsStyle='primary'>
-                    <Icon type='wheelchair-alt' withSpace />
-                    <FormattedMessage id='components.TripViewer.accessible' />
+                {tripData.wheelchairAccessible === 1 && (
+                  <Label bsStyle="primary">
+                    <Icon type="wheelchair-alt" withSpace />
+                    <FormattedMessage id="components.TripViewer.accessible" />
                   </Label>
-                }
+                )}
                 <SpanWithSpace margin={0.25} />
-                {tripData.bikesAllowed === 1 &&
-                  <Label bsStyle='success'>
-                    <Icon type='bicycle' withSpace />
-                    <FormattedMessage id='components.TripViewer.bicyclesAllowed' />
+                {tripData.bikesAllowed === 1 && (
+                  <Label bsStyle="success">
+                    <Icon type="bicycle" withSpace />
+                    <FormattedMessage id="components.TripViewer.bicyclesAllowed" />
                   </Label>
-                }
+                )}
               </h4>
             </div>
           )}
 
           {/* Stop Listing */}
-          {tripData && tripData.stops && tripData.stopTimes && (
+          {tripData &&
+            tripData.stops &&
+            tripData.stopTimes &&
             tripData.stops.map((stop, i) => {
               // determine whether to use special styling for first/last stop
               let stripMapLineClass = 'strip-map-line'
               if (i === 0) stripMapLineClass = 'strip-map-line-first'
-              else if (i === tripData.stops.length - 1) stripMapLineClass = 'strip-map-line-last'
+              else if (i === tripData.stops.length - 1)
+                stripMapLineClass = 'strip-map-line-last'
 
               // determine whether to show highlight in strip map
               let highlightClass
-              if (i === viewedTrip.fromIndex) highlightClass = 'strip-map-highlight-first'
-              else if (i > viewedTrip.fromIndex && i < viewedTrip.toIndex) highlightClass = 'strip-map-highlight'
-              else if (i === viewedTrip.toIndex) highlightClass = 'strip-map-highlight-last'
+              if (i === viewedTrip.fromIndex)
+                highlightClass = 'strip-map-highlight-first'
+              else if (i > viewedTrip.fromIndex && i < viewedTrip.toIndex)
+                highlightClass = 'strip-map-highlight'
+              else if (i === viewedTrip.toIndex)
+                highlightClass = 'strip-map-highlight-last'
```

Comment:
I would use brackets here.

Reply:
Good catch thanks! Prettier's doing

fixed in ca08deb